addons/point_of_sale/tests/test_pos_other_currency_config.py

```diff
@@ -288,4 +288,4 @@ def test_05_tax_base_amount(self):
         tax_line = session_account_move.line_ids.filtered(lambda line: line.account_id == self.tax_received_account)
         self.assertAlmostEqual(tax_line.balance, -3.44)
         self.assertAlmostEqual(tax_line.amount_currency, -1.72)
-        self.assertAlmostEqual(tax_line.tax_base_amount, 49, msg="Value should be in company's currency.")
+        self.assertAlmostEqual(tax_line.tax_base_amount, -49, msg="Value should be in company's currency.")
```

Comment:
Looking at the tests in `account`, it appears that `tax_base_amount` is always positive. I also checked by creating invoice and credit notes, and regardless of whether the tax value is credit or debit, the `tax_base_amount` is positive.

Perhaps instead of changing the test, we just take the absolute value of `tax_base_amount` in https://github.com/odoo/odoo/blob/c95fad77ea5b4663130e256f76031560171f09e6/addons/point_of_sale/models/pos_session.py#L686